Dockerfile-Server

```diff
@@ -0,0 +1,49 @@
+FROM centos:7
+
+WORKDIR /dockermount
+
+RUN yum -y update && yum clean all
+RUN yum -y install wget gcc openssl-devel bzip2-devel libffi libffi-devel zlib-devel
+RUN yum -y groupinstall "Development Tools"
+
+##### Install PostgreSQL 10 client (psql)
+RUN yum -y install https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+RUN yum -y install postgresql10
+
+##### Building python 3.7
+WORKDIR /usr/src
+RUN wget --quiet https://www.python.org/ftp/python/3.7.3/Python-3.7.3.tgz
+RUN tar xzf Python-3.7.3.tgz
+WORKDIR /usr/src/Python-3.7.3
+RUN ./configure --enable-optimizations
+RUN make altinstall
+RUN ln -sf /usr/local/bin/python3.7 /usr/bin/python3
+RUN echo "$(python3 --version)"
+
+##### Copy python packaged
+WORKDIR /dockermount
+COPY requirements/ /dockermount/requirements/
+RUN python3 -m pip install -r requirements/requirements.txt
+
+##### Additional server requirements
+RUN yum install -y \
+    git \
+    gcc-c++ \
+    postgresql-devel \
+    pcre \
+    pcre-devel \
+    mlocate \
+    wget \
+    unzip \
+    bzip2-devel \
+    memcached \
+    libmemcached \
```

Comment:
I'm still not sure where and why `memcached` becomes a dependency to usaspending. This may be a stale holdover from early days _before_ switching to a Redis-based caching system in AWS elasticache.

Reply:
these are dependencies to being able to install `pylibmc` pip package which is in the `requirements-server.txt` file.
If we can safely remove this application code dependency, we can also remove these two yum packages.

Reply:
resolved as part of latest patch